example/storybook/stories/TrackTile/util/trackerData.ts

```diff
@@ -0,0 +1,146 @@
+import {
+  Tracker,
+  TRACKER_CODE_SYSTEM,
+  TRACKER_PILLAR_CODE_SYSTEM,
+} from '../../../../../src/components/TrackTile/services/TrackTileService';
+
+export const getTrackers = ({
+  names,
+  pillars,
+}: {
+  names?: string[];
```

Comment:
It would be nice if this was typed. I also don't see where this property is used.

Reply:
Oh, yes, this was used in the next PR but it was removed and simplified.  I need to remove it here too since it isn't currently used anymore.

Reply:
Removed.